### PR TITLE
Added a /swagger endpoint to access swagger UI.

### PR DESCRIPTION
**Pull request by cowmanjoe**
Swagger uses the OpenAPI spec to display API documentation in a clean and interactive way. The server now uses Swagger to do just this. Just access the /swagger endpoint in a browser to test it out! An instance of our Postgres database also needs to be running on your machine for this to work. 

The server/swagger.json file contains all the information that Swagger uses to display the API. If we keep using this, any new API endpoint should be documented in this file. OpenAPI is relatively simple to use, and there is plenty of documentation online for it.

### CODE REVIEW

File: server/controllers/mailingListController.ts [hunk 1]
```diff
@@ -7,6 +7,10 @@ export async function addToMailingList(req: Request, res: Response): Promise<voi
     let connection: Connection = getConnection();
 
     let email: string = req.query.email;
+
+    if ((typeof email) !== "string") {
+        res.status(400).send({error: "Email address query parameter is required"});
```
Comment (cowmanjoe) [inline server/controllers/mailingListController.ts:15]: This is unrelated to the PR but a bug I discovered as I was trying to get it working.

File: server/swagger.json [hunk 2]
```diff
@@ -0,0 +1,271 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This is the API for Internado's back end. It's main function right now is searching for other internships in the database. Check out our [git repo](https://github.com/ubclaunchpad/Internado) or our [wiki](https://github.com/ubclaunchpad/Internado/wiki) for more information.",
```
Comment (frostyshadows) [inline server/swagger.json]: should be `Its main function` instead of `It's main function`

**Review by frostyshadows — CHANGES_REQUESTED** [review]
Could you also add info on how to access the Swagger page on the main README?

**Review by frostyshadows — APPROVED** [review]
LGTM 🌪

**Pull request merged**